kurtosis-devnet/pkg/kurtosis/endpoints.go

```diff
@@ -269,9 +269,9 @@ func (f *ServiceFinder) triage() {
 		// We should expect the rules to be gradually removed to make way for
 		// this code path. Ultimately we'll rely only on labels, and most of the
 		// code in this file will disappear as a result.
-		triaged := rules.apply(serviceName, endpoints)
+		triaged := f.triageByLabels(svc, serviceName, endpoints)
```

Comment:
the comment above should be cleaned up a little bit, as labels are promoted to main identification device.

Reply:
done